docs/guides/sparque-integration.md

```diff
@@ -0,0 +1,84 @@
+<!--
+kb_guide
+kb_pwa
+kb_everyone
+kb_sync_latest_only
+-->
+
+# SPARQUE.AI Integration
+
+ SPARQUE.AI works as an AI-powered search engine and delivers various information, like keyword suggestions, search results, filter options, and category navigation.
```

Comment:
I added AI-powered here to mention its unique feature. If the following is also true (I wasn't sure), and if you like it better, you could rephrase it to:

SPARQUE.AI works as a search engine and delivers various information, like keyword suggestions, search results, filter options, and category navigation based on AI-driven algorithms.